project.yaml

```diff
@@ -673,21 +675,21 @@ Layer:
     properties:
       minzoom: 14
     advanced: {}
-  - id: "cliffs"
-    name: "cliffs"
+  - id: "reliefsymbols"
+    name: "reliefsymbols"
     class: ""
     geometry: "linestring"
     <<: *extents
     Datasource:
       <<: *osm2pgsql
       table: |-
         (SELECT
-            way, "natural", man_made
+            way, "natural", name, ST_Length(way)/NULLIF(SQRT(!pixel_width!::real*!pixel_width!::real),0) AS pixel_length, man_made
```

Comment:
`pixel_length` is NULL for zoom level 15 at 43°30'. a) is this right? b) shouldn't it be `!pixel_width!::real*!pixel_height!::real`